xinference/api/restful_api.py

```diff
@@ -1344,9 +1344,9 @@ async def create_chat_completion(self, request: Request) -> Response:
                     status_code=400,
                     detail=f"Only {function_call_models} support tool messages",
                 )
-        if body.tools and body.stream:
+        if body.tools and body.stream and not model.is_vllm_backend():
             raise HTTPException(
-                status_code=400, detail="Tool calls does not support stream"
+                status_code=400, detail="Tool calls only support streaming when using VLLM"
```

Comment:
Why tool call `only` supports streaming in this PR?

Reply:
There seems to be an ambiguity. What about "Streaming support for tool calls is available only when using vLLM."